ldap/client.go

```diff
@@ -273,7 +273,11 @@ func (c *Client) Authenticate(ctx context.Context, username, password string, op
 			return nil, fmt.Errorf("%s: failed to get user attributes: %w", op, err)
 		}
 		for _, a := range attrs {
-			userAttrs[a.Name] = a.Vals
+			name := a.Name
```

Comment:
Mind adding WithLowerUserAttributeKeys to the list of supported optins in the
func's godoc?

Reply:
Good call, done